tests/test_workers/test_config.py

```diff
@@ -144,3 +144,32 @@ def test_validate_celery_config_invalid_organization_customizations(config, erro
     }
     with pytest.raises(ConfigError, match=error):
         validate_celery_config(conf)
+
+
+def test_validate_celery_config_request_logs_dir_missing():
+    missing_path = '/this/path/does/not/exist'
```

Comment:
Could you mock `os.path.exists` or use a path within `tmpdir` instead of this? I don't like the tests reaching out on the file system outside of the Git repo and `/tmp`.

Reply:
I can't get this check to work if the directory is created by python. https://docs.python.org/3/library/os.html#os.access

There's also a security warning there.

There's value in doing this check though. I'll dig more.

Reply:
Ha! It's because I was running the tests with sudo and we can't restrict access of the root user. I'll add a conditional skip test in there.